Source/santasyncservice/SNTSyncFCM.m

```diff
@@ -212,24 +203,13 @@ - (void)reachabilityRestored {
 
 /**  Start listening for network state changes on a background thread. */
 - (void)startReachability {
-  if (self.reachability) return;
   LOGD(@"Reachability started.");
-  self.reachability =
-    SCNetworkReachabilityCreateWithName(kCFAllocatorDefault, _connectComponents.host.UTF8String);
-  SCNetworkReachabilityContext context = {.info = (__bridge void *)self};
-  if (SCNetworkReachabilitySetCallback(self.reachability, reachabilityHandler, &context)) {
-    SCNetworkReachabilitySetDispatchQueue(
-      self.reachability, dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_BACKGROUND, 0));
-  }
+  nw_path_monitor_start(self.pathMonitor);
```

Comment:
Do we know what happens if `nw_path_monitor_start` is called multiple times?

Reply:
Good question, it's not documented anywhere so I tested and it turns out the monitor starts multiple times :-(

I've updated the code to only create the monitor in startReachability and to cancel it out in stopReachability